corehq/apps/ota/views.py

```diff
@@ -184,7 +184,8 @@ def _add_case_property_queries(domain, case_type, search_es, criteria):
             case_property=key,
         )
         for removal_regex in remove_char_regexs:
-            value = re.sub(removal_regex.regex, '', value)
+            to_remove = re.escape(removal_regex.regex)
```

Comment:
@sravfeyn i see, maybe this requires input from product then on whether this should be treated as a regex or substring match